src/styles/card.css

```diff
@@ -0,0 +1,98 @@
+/* region card */
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-image {
+  width: 160px;
+  height: 134px;
+
+  margin-bottom: 40px;
+}
+
+.card__product-name {
+  margin-bottom: 4px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card_product-code {
+  margin-bottom: 16px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__product-rating {
+  width: 100%;
+  margin-bottom: 24px;
+
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.card_product-review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__product-cost {
+  width: 100%;
+  margin-bottom: 16px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.cars_product-price {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.cars_product-price-num {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__product-buy {
+  height: 40px;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  background: #00acdc;
+  border-radius: 5px;
+
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  color: #fff;
+  text-decoration: none;
+  text-transform: uppercase;
+
+  border: 1px solid transparent;
+}
+
+.card__product-buy:hover {
+  background-color: #fff;
+  border-color: #00acdc;
+  color: #00acdc;
+}
```

Comment:
Use variables (in the entire code)

Reply:
added variable for color, but I didn`t see anything in CL about using variables

Reply:
Misha talked about this in his video lectures